src/main/java/core/basesyntax/dao/FruitDaoImpl.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.Storage;
+
+public class FruitDaoImpl implements FruitDao {
+    @Override
+    public void add(String fruit, int quantity) {
+        Storage.report.put(fruit, quantity);
+    }
+
+    @Override
+    public Integer get(String fruit) {
+        return Storage.report.get(fruit);
+    }
```

Comment:
```suggestion
    private final Storage storage = new Storage();

    @Override
    public void add(String fruit, int quantity) {
        storage.add(fruit, quantity);
    }

    @Override
    public Integer get(String fruit) {
        return storage.getQuantity(fruit);
    }
```

Reply:
To keep DAO class while having storage map as private, we can do smth like this, adjust to your needs